lesson6Khanh/src/main/java/com/trainingdeveloperpro/androidk001/lesson5/nguyenquockhanh/studentlist/StudentAdapter.kt

```diff
@@ -0,0 +1,54 @@
+package com.trainingdeveloperpro.androidk001.lesson5.nguyenquockhanh.studentlist
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.recyclerview.widget.DiffUtil
+import androidx.recyclerview.widget.RecyclerView
+import android.os.Handler
+
+class StudentAdapter(private var items: ArrayList<Student>) : RecyclerView.Adapter<StudentViewHolder>() {
+    override fun getItemCount(): Int {
+        return items.size
+    }
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): StudentViewHolder {
+        val inflater = LayoutInflater.from(parent.context)
+        return StudentViewHolder(inflater, parent)
+    }
+
+    override fun onBindViewHolder(holder: StudentViewHolder, position: Int) {
+        val student: Student = items[position]
```

Comment:
Đoạn này Kotlin cho phép không cần khai báo kiểu
```suggestion
        val student = items[position]
```